lib/core/db_models/old/schema/staff_professional_detail_schema.dart

```diff
@@ -2,6 +2,6 @@ class StaffProfessionalDetailSchema {
   static const String skills = "skills";
   static const String checks = "checks";
   static const String qualifications = "qualifications";
-  static const String workHistory = "work_history";
+  static const String workHistories = "work_histories";
```

Comment:
guessing this is a mistake and we shouldn't have an underscore between these two words

Reply:
its a key name which is coming from api, the variable name does not have any underscore